### ISSUE TITLE
Multi param method calls indent doesn't match Xcode

### ISSUE THREAD

**Issue by szweier** [nicklockwood/SwiftFormat#42]
Xcode indent:
```
let result = self.method(value1,
                         param2: value2,
                         param3: value3)
```

SwiftFormat:
```
let result = self.method(value1,
    param2: value2,
    param3: value3)
```

Is there a setting to keep the format matching Xcode for this case?

**Comment by nicklockwood** [comment]
Not currently, but I'll consider adding support for this.

**Comment by nicklockwood** [comment]
In the meantime, you may find the formatting looks nicer when written like this:

    let result = self.method(
        value1,
        param2: value2,
        param3: value3
    )

**Comment by ollitapa** [comment]
I would like to see colon alignment like this:
```swift
let result = self.method(param1: value1,
                         param2: value2,
                      param3333: value3)
```
I'm not sure if this is the swift way, but this used to make code look cleaner in Obj-C

**Comment by nicklockwood** [comment]
@szweier this is fixed in 0.14

(@ollitapa colon alignment isn't supported at the moment, but feel free to open a separate feature request for that)
